exporter/awscloudwatchlogsexporter/exporter.go

```diff
@@ -72,7 +72,7 @@ func newCwLogsPusher(expConfig *Config, params exp.CreateSettings) (*exporter, e
 	}
 
 	// create CWLogs client with aws session config
-	svcStructuredLog := cwlogs.NewClient(params.Logger, awsConfig, params.BuildInfo, expConfig.LogGroupName, expConfig.LogRetention, session)
+	svcStructuredLog := cwlogs.NewClient(params.Logger, awsConfig, params.BuildInfo, expConfig.LogGroupName, expConfig.LogRetention, session, false)
```

Comment:
nit: Creating a withOption pattern might be cleaner so we dont change usages such as this

Reply:
I will take a look at that

Reply:
I am going to leave this as is.  This doesn't really save any code because only 2 of these are even member variables.  I would have to introduce member variables for all of these so they can be used to construct the client.